aslam_backend_expressions/include/aslam/backend/implementation/VectorExpression.hpp

```diff
@@ -18,7 +18,12 @@ namespace aslam {
       _root(new ConstantVectorExpressionNode<D>(v))
     {
     }
-      
+
+    template <int D>
+    template <typename... Args>
+    VectorExpression<D>::VectorExpression(const ScalarExpression& expr, Args&&... remainingExprs)
+        : _root(new StackedScalarVectorExpressionNode<D>(expr._root, remainingExprs._root...)) {}
```

Comment:
I'm not sure this is the right place to to this "transformation". Usually the operator implemented in the expression would do this and since we dont have that, this felt like the closest to it place